serverless.yml

```diff
@@ -27,11 +33,20 @@ custom:
     subnetIds: ${ssm:/iow/aws/vpc/${self:provider.stage}/subnetIds~split}
   db:
     connectInfo: ${ssm:/aws/reference/secretsmanager/NWCAPTURE-DB-${self:provider.stage}~true}
+  esLogs:
+    endpoint: ${ssm:/iow/aws/vpc/${self:provider.stage}/elasticsearchEndpoint}
+    index: ${self:service}
+    retentionInDays: 90
+    filterPattern: '?Exception ?ERROR ?WARN'
```

Comment:
I thought it was ?ERROR ?CRITICAL elsewhere ...

Reply:
That's a good point... I erred for WARN and up.  Should this actually be something like `'?Exception ?ERROR ?WARN ?FATAL'?

Reply:
Maybe it was a python vs java thing.  I don't know what the options are, I just saw a (potential) inconsistency if some things are passing error + critical and some are passing error + warn.

Reply:
`?ERROR ?CRITICAL` is being used for the python lambdas.